README.md

```diff
@@ -1,2 +1,101 @@
 # gulp-testcafe
-A Gulp plugin for TestCafe. 
+[![Build status](https://ci.appveyor.com/api/projects/status/gvjohww3gs65m36j?svg=true)](https://ci.appveyor.com/project/DevExpress/gulp-testcafe)
+
+*Run TestCafe tests using Gulp.*
+
+## Install
+```sh
+npm install --save-dev gulp-testcafe
+```
+
+## Usage
+```js
+const gulp     = require('gulp');
+const testcafe = require('gulp-testcafe');
+
+gulp.task('default', () => {
+    return gulp.src('test.js')
+        .pipe(testcafe({ browsers: ['chrome', 'firefox'] }));
+});
+```
+
+## API
+```js
+testcafe(options)
+```
+
+### Options
+
+#### browsers
+
+
+*Type*: `Array`
+
+*Default*: `[]`
+
+*Values*: [Specifying Browsers for Test Task](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/browser-support.html#specifying-browsers-for-test-task)
+
+*Required*
+
+Configures the test runner to run tests in the specified browsers.
+
+#### reporter
+
+*Type*: `String`
+
+*Default*: `spec`
+
+*Values*: [Reporters](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/reporters.html)
+
+Specifies the reporter.
+
+#### filter
+
+*Type*: `function(testName, fixtureName, fixturePath)`
+
+*Default*: `null`
+
+Allows you to manually select which tests should be run.
```

Comment:
Let's add more references to **Programming interface** from here.

Probably, add a section after the descriptions like

```
*Details*: (link)
```